spec.html

```diff
@@ -28168,7 +28168,7 @@ <h1>Math.acosh ( _x_ )</h1>
 
       <emu-clause id="sec-math.asin">
         <h1>Math.asin ( _x_ )</h1>
-        <p>Returns an implementation-approximated value representing the result of the arc sine of _x_. The result is expressed in radians and ranges from -&pi; / 2 to +&pi; / 2.</p>
+        <p>Returns the inverse sine of _x_. The argument is expressed in radians and ranges from -&pi; / 2 to +&pi; / 2.</p>
```

Comment:
You changed a lot of these to "the argument", but I think they're supposed to be talking about the result.

Reply:
My mistake! Got a little carried away in the standardization. :/

Reply:
Fixed this up!